x-pack/performance/journeys/apm_service_inventory.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Journey } from '@kbn/journeys';
+import { SynthtraceClient } from '../services/synthtrace';
+import { generateData } from '../synthtrace_data/apm_data';
+
+export const journey = new Journey({
+  beforeSteps: async ({ kbnUrl, log, auth, es }) => {
+    // Install APM Package
+    const synthClient = new SynthtraceClient({
+      kbnBaseUrl: kbnUrl.get(),
+      logger: log,
+      username: auth.getUsername(),
+      password: auth.getPassword(),
+      esClient: es,
+    });
+
+    await synthClient.installApmPackage();
+    // Setup Synthtrace Client
+    await synthClient.initialiseEsClient();
+    // Generate data using Synthtrace
+    const start = Date.now() - 1000;
+    const end = Date.now();
+    await synthClient.index(
+      generateData({
+        from: new Date(start).getTime(),
+        to: new Date(end).getTime(),
+      })
+    );
+  },
```

Comment:
```suggestion
  },
  ftrConfigPath: 'x-pack/performance/configs/apm_config.ts',
```
otherwise journey will use the default one

Reply:
Ah good catch, didn't realise